docs/application/dotnet/guides/applications/service_application.md

```diff
@@ -0,0 +1,245 @@
+# Service Applications 
```

Comment:
Since this is a new topic, it need to be included in the overview topic :(https://docs1.stg.tizen.org/staging/1494/application/dotnet/guides/applications/overview/)

Kindly check and update overview topic and include in the PR.